Mathlib/Topology/Support.lean

```diff
@@ -356,9 +356,11 @@ theorem HasCompactSupport.mul_left (hf : HasCompactSupport f') : HasCompactSuppo
 
 end MulZeroClass
 
-namespace LocallyFinite
+section LocallyFinite
+
+variable {ι : Type*} [TopologicalSpace X]
 
-variable {ι : Type*} {U : ι → Set X} [TopologicalSpace X] [One R]
+namespace LocallyFinite
```

Comment:
Since there is only one lemma, can you just not open the namespace, and add it to the lemma name?